### DISCUSSION TITLE
Customise password display?

### DISCUSSION THREAD

**Discussion by djclark46** [lithnet/access-manager#88] [Q&A]
Hi,

We have a user with Dyslexia who is finding the way the passwords are displayed challenging:

"Basically I have dyslexia and the dual colour combination used on the ams website for the LAPS passwords makes it incredibly hard for me to actually read the password as it's visually challenging for me. The font is fine it's just the colour usage. "

Is there any way to customise how the passwords are displayed?

Thanks

Dave

**Comment by ryannewington** [comment]
Hi Dave,

We can definitely build something in to support this case. Would a toggle button on the password page to strip the colour be suitable?

Are there any other changes we could look at to assist further?

**Reply by djclark46** [reply]
Hi Ryan,

The user said that would be a great help and she's also going to have a think if there's anything else that would help.

Dave

**Comment by ryannewington** [comment]
@djclark46 

I've added some display options to the UI. The image below shows the standard password display, and then the same password with monochrome, large font, and increased space 'modifiers' turned on. The settings are saved per browser for the user, so they don't have to select them every time.

If any combination of these options are suitable for your user, I'll get you a new build to try.

![image](https://user-images.githubusercontent.com/17229877/113647839-eab39880-96ce-11eb-8502-959b56c1511f.png)

**Reply by djclark46** [reply]
Hi Ryan,

She says thats much better thanks, monochrome and larger font. Can you get me the new build to try?

Thanks

Dave

**Reply by ryannewington** [reply]
@djclark46 The release is published here https://github.com/lithnet/access-manager/releases/tag/1.0.7888.0-rc2

**Reply by djclark46** [reply]
Ryan, unfortunately the options dont work in firefox and IE (cant select the drop downs). Works ok in chrome, Chrome Edge, Beta Chrome Edge so plenty of options but thought you'd like to know. Running 87.0 64 Bit firefox. I dont care about IE :)

**Reply by ryannewington** [reply]
That's odd! Firefox is my main dev browser. Let me look into this a bit more.

IE isn't something we support with AMS. It kinda works, but certainly not everything. 

I'll get back to you on the Firefox issue shortly

**Reply by ryannewington** [reply]
@djclark46 I can't reproduce this. Could there be a plugin of some sort that is blocking javascript in your firefox browser?

What are you actually seeing? Is the drop down not visible, does it not drop down, etc

**Reply by djclark46** [reply]
Drop down comes up but I cant select any options (wont let me select any - I can see them. Screenshot added.
![2021-04-15](https://user-images.githubusercontent.com/61976271/114912480-f2ceaf00-9e17-11eb-923f-85aa755fd581.png)

**Reply by ryannewington** [reply]
That not even displaying in the correct spot. Can you CTRL+F5 or clear the browser cache to reload the page and see if that fixes it?

**Reply by djclark46** [reply]
CTRL + F5 fixed it, thanks :) It also now displays on the rhs.


Dave
